spec/classes/init_spec.rb

```diff
@@ -293,6 +296,7 @@
               value: '10000',
             )
           }
+          it { is_expected.to have__loginctl_user('foo').with(linger: 'enabled') }
```

Comment:
I think this fixes it:
```suggestion
          it { is_expected.to contain__loginctl_user('foo').with(linger: 'enabled') }
```